### PR TITLE
Add eval option for cafe

### LINKED ISSUE #54: Is there a way to check the accuracy of each result?
It would be very convenient if the accuracy could be shown.

### PR DESCRIPTION
**Pull request by marinelay**
close #54

### CODE REVIEW

**Comment by henrylee97** [comment]
What does this PR do?

**Comment by marinelay** [comment]
I added `eval` option for cafe framework and further tools.
You can see the evaluation by the following command: `kaprese eval -e <engine>`.
It is a prototype for the full support, so if you accept this modification, I can develop more detailed points.

File: kaprese/utils/const.py [hunk 1]
```diff
@@ -0,0 +1 @@
+OUTPUT_DIRECTORY = "kaprese-out"
```
Comment (henrylee97) [inline kaprese/utils/const.py:1]: How about using arguments like `run` command?

See:
https://github.com/kupl/kaprese/blob/0d40cdd2191077cb1fc360800d56bd81516c11c4/kaprese/bin/kaprese_run.py#L285-L290

**Comment by henrylee97** [comment]
I think you should clarify the operations implemented clearly and reference other files to line up the implementation styles.

**Pull request closed without merging**